websites/permissions.py

```diff
@@ -0,0 +1,163 @@
+""" permissions for websites """
+from django.contrib.auth.models import Group, Permission
+from django.db import transaction
+from guardian.shortcuts import assign_perm
+from rest_framework.generics import get_object_or_404
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+from websites import constants
+from websites.models import Website
+
+
+def assign_object_permissions(group_or_user, website, perms):
+    """
+    Assign appropriate permissions to a website admin group
+
+    Args:
+        group_or_user (Group or User): the group/user to add permissions to
+        website (Website): The website that the permissions are for
+        perms (list of str): The permissions to be assigned
+    """
+    for perm in perms:
+        try:
+            assign_perm(perm, group_or_user, website)
+        except Permission.DoesNotExist as err:
+            raise Permission.DoesNotExist(f"Perm {perm} not found") from err
```

Comment:
Is the other exception message not useful?

Reply:
The default message doesn't indicate what the permission is:
`Permission matching query does not exist.` vs `Perm fakePermission not found`